releases/CHANGELOG-v0.80.1.md

```diff
@@ -0,0 +1,9 @@
+
+<a name="v0.80.1"></a>
+# [v0.80.1] - 2024-01-24
```

Comment:
Could we call out the taint change + new azuredisk csi driver version in a new #attention section. 

See 0.79.0 release notes for example: https://github.com/Azure/aks-engine-azurestack/pull/207/files